packages/terra-core-docs/src/terra-dev-site/doc/search-field/About.1.doc.mdx

```diff
@@ -1,35 +1,90 @@
+|[SearchFieldWithCustomLabel](/components/cerner-terra-core-docs/search-field/examples/search-field-with-custom-label)|An example demonstrating search field with a custom label.|
+|[SearchFieldPlaceholder](/components/cerner-terra-core-docs/search-field/examples/search-field-placeholder)|An example demonstrating search field with a placeholder.|
+|[SearchFieldMinimumLength](/components/cerner-terra-core-docs/search-field/examples/search-field-minimum-length)|An example demonstrating search field with a minimum search text length set.|
+|[SearchFieldDelayed](/components/cerner-terra-core-docs/search-field/examples/search-field-delayed)|An example demonstrating search field with a search delay.|
+|[SearchFieldDefaultValue](/components/cerner-terra-core-docs/search-field/examples/search-field-default-value)|An example demonstrating search field with a default search text value.|
+|[SearchFieldDisabled](/components/cerner-terra-core-docs/search-field/examples/search-field-disabled)|An example demonstrating disabled search field.|
+|[SearchFieldDisabledAutoFormatAttrs](/components/cerner-terra-core-docs/search-field/examples/search-field-disable-auto-format-attrs)|An example demonstrating search field with disabled autoformat attributes.|
+|[SearchFieldBlock](/components/cerner-terra-core-docs/search-field/examples/search-field-block)|An example demonstrating search field with block style|
+|[SearchFieldDisableAutoSearch](/components/cerner-terra-core-docs/search-field/examples/search-field-disable-auto-search)|An example demonstrating search field with autosearch disabled.|
+|[SearchFieldFilterNumeric](/components/cerner-terra-core-docs/search-field/examples/search-field-filter-numeric)|An example demonstrating search field with numeric filtering.|
+|[SearchFieldFocus](/components/cerner-terra-core-docs/search-field/examples/search-field-focus)|An example demonstrating search field with focus.|
+|[SearchFieldProgrammaticSet](/components/cerner-terra-core-docs/search-field/examples/search-field-programmatic-set)|An example demonstrating search field with search text set programmatically.|
+
+
+## Accessibility
+### Keyboard Interactions
+
+List of keyboard interactions supported by Search Field
+
+|Key interactions|Description|
+|---|---|
+|Tab           | Moves focus into the search field. If already on the search text box with text entered, tab moves focus to the clear text button. If already on clear text button, tab moves followed by focus to the search button.|
+|Shift + Tab    | Moves focus to the previous focusable element in the tab sequence.|
+|Space| When on clear button, clears the current search text. When on search button, triggers the search callback.|
+|Enter| When on clear button, clears the current search text. When on search button, triggers the search callback.|
```

Comment:
@chrismichalewicz  Do you agree that we need not have this. Making sure this is okay since the template you shared had the Keyboard interactions section.

Reply:
My preference is to include all keyboard interactions supported by the component. This is what ARIA APG site does, and it helps reinforce to consumers who may not be familiar with accessibility how things are supposed to work.

Reply:
@nikhitasharma  I agree with @mjpalazzo  I think it is good to include all support keyboard interactions even if we think they are common.  Even so we are making an assumption that all users know those commands and that might not be the case.